internal/testutils/xds/e2e/setup_management_server.go

```diff
@@ -41,7 +41,7 @@ import (
 // - bootstrap contents to be used by the client
 // - xDS resolver builder to be used by the client
 // - a cleanup function to be invoked at the end of the test
-func SetupManagementServer(t *testing.T, opts *ManagementServerOptions) (*ManagementServer, string, []byte, resolver.Builder, func()) {
+func SetupManagementServer(t *testing.T, opts ManagementServerOptions) (*ManagementServer, string, []byte, resolver.Builder, func()) {
```

Comment:
Why do we want to switch this? For the future to not have nil checks?

Reply:
Yes. And the sooner we switch the better it is, so that we don't keep adding new `nil` checks.